src/ContentHubClient.php

```diff
@@ -1157,4 +1159,20 @@ protected static function removeAllLeadingAndTrailingSlashes(array $components):
             return trim($component, '/');
         }, $components);
     }
+
+    /**
+     * @param array $config
+     */
+    protected function enableRequestLogging(array $config): void
```

Comment:
Seems like enableRequestLogging is a misnomer. As the code indicates, it does not enable logging, but instead it adds a request logger handler to the stack.

Reply:
Renamed